### DISCUSSION TITLE
Switching between workspaces broken

### DISCUSSION THREAD

**Discussion by shadow-magic** [i3/i3#4823] [Q&A]
Using i3 on Endeavour OS an arch distro. i3 version 4.20.1
When I switch to another workspace it will often show me the workspace i just came from. If I try to switch an application to that workspace, one half of the screen will be occupied by whatever workspace i just came from, and the other half by the application that I just moved there. For example, if I came from an empty workspace, half of the screen will show the wallpaper, and if i just came from firefox, that half will show firefox. Please advise.

https://logs.i3wm.org/logs/6261493226012672.bz2

**Comment by Airblader** [comment]
Since you chose not to fill out the issue template for bugs I assume you're not trying to report a bug. Converting this into a discussion. In the future please do not ignore issue templates, many projects will simply close your issue for that reason alone.

This issue is typically graphics related. If you're using a compositor, try disabling it. Make sure your graphics drivers are you to date.
